docs/installation/presto-configuration.rst

```diff
@@ -46,5 +46,33 @@ to use a lower value for ``query.max-memory-per-node``.
 If you are running Presto in a test environment that has less than 16 GB of memory available,
 you will need to follow similar procedures to set the memory configurations lower.
 
+Other properties that should be updated for a production environment are the log locations. In order to update these:
```

Comment:
Can you make a sub-header for both memory and log locations?

Also, Suggest:
For most production environments, it will be necessary to change the log location.